templates/web.template.yml

```diff
@@ -203,24 +191,6 @@ run:
       hook: plugin_compatibility
       raise_on_fail: false
 
-  - exec:
-      cd: $home
-      tag: migrate
-      hook: db_migrate
-      cmd:
-        - su discourse -c 'bundle exec rake db:migrate'
-  - exec:
-      cd: $home
-      tag: build
-      hook: assets_precompile_build
-      cmd:
-        - su discourse -c 'bundle exec rake assets:precompile:build'
-  - exec:
-      cd: $home
-      tag: precompile
-      hook: assets_precompile
-      cmd:
-        - su discourse -c 'SKIP_EMBER_CLI_COMPILE=1 bundle exec rake themes:update assets:precompile'
```

Comment:
Just want to make sure that we are not removing these intentioanlly?

Reply:
this doesn't look like it should be removed from the template, my guess was this was temporarily removed for testing the nginx template updates @nbianca ?

Reply:
Oh, that's right. My bad. I will add them back. :)